store/migrations/000017_create_post_hotness.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE post_hotness (
```

Comment:
Totally a nit but I'd like to see us avoiding the term "hotness" and maybe just lean towards "score" - we might end up creating algos which don't target "popularity".